docs/content/guide/migration.ngdoc

```diff
@@ -30,6 +31,967 @@ which drives many of these changes.
+- Implementing long awaited features, such as support for inputs of type `range` or the ability to
+  bind to any type of values using `ngRepeat` with `select`.
+- Disabling (by default) the pre-assignment of bindings on controller instances.
+- Changing the default `$location` hash-prefix to `!`.
+- Reporting possibly unhandled promise rejections that would otherwise go unnoticed.
+
+Another major change is the removal of the Expression Sandbox. This should not affect applications
+(other than a very slight performance boost), but we strongly recommend reading the
+[Sandbox Removal Blog Post](http://angularjs.blogspot.com/2016/09/angular-16-expression-sandbox-removal.html)
+to understand the implications behind the removal and whether any action is required on your part.
+
+
+You may also notice that this release comes with a longer-than-usual list of breaking changes. Don't
+let this dishearten you though, since most of them are pretty minor - often not expected to affect
+real applications. These breaking changes were necessary in order to:
+
+- Align with breaking changes in jQuery 3.
+- Fix bugs that we wouldn't be able to fix otherwise.
+- Introduce new features, performance improvements and security fixes.
+- Make the behavior of existing features more consistent and predictable.
+
+
+To give you a heads-up, here is a brief summary of the breaking changes that are expected to have
+the highest impact. Make sure you look them up in the full list below or check out the corresponding
+commits for more info.
+
+- **input[type=range]**: Due to [913016](https://github.com/angular/angular.js/commit/9130166767c4792c5d32d08a918fc7becf32c9a6)
+  and the built-in support for range inputs, the behavior of such elements when bound to `ngModel`
+  will be different than before.
+
+- **ngModelOptions**: Due to [87a2ff](https://github.com/angular/angular.js/commit/87a2ff76af5d0a9268d8eb84db5755077d27c84c),
+  if a setting is not applied on `ngModelOptions`, it will be inherited from the nearest
+  `ngModelOptions` ancestor. Previously, it would default to `undefined`.
+  The programmatic API for `ngModelOptions` has also changed.
+
+- **ngTransclude**: Due to [32aa7e](https://github.com/angular/angular.js/commit/32aa7e7395527624119e3917c54ee43b4d219301),
+  if you only provide whitespace as the transclusion content, it will be assumed to be empty and the
+  fallback content will be used instead. Previously, whitespace only transclusion would be treated as
+  the transclusion being "not empty", which meant that fallback content was not used in that case.
+
+- **$location**: Due to [aa077e8](https://github.com/angular/angular.js/commit/aa077e81129c740041438688dff2e8d20c3d7b52),
+  the default hash-prefix used for `$location` hash-bang URLs has changed from the empty string
+  (`''`) to the bang (`'!'`).
+
+- **jqLite**:
+    - Due to [fc0c11](https://github.com/angular/angular.js/commit/fc0c11db845d53061430b7f05e773dcb3fb5b860),
```

Comment:
So in this one it could look something like:

> * jqLite is more aligned to jQuery 3 which required the following changes (see [link to section]):
>   * keys for `.data()` and `.css()` are now camelCased
>   * setting boolean attributes to empty string no longer removes the attribute
>   * calling `.val()` on a multiple select will now return an empty array if no option is selected